<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Playfair</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="T_sifrovat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Sifrovat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="T_desifrovat">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Desifrovat</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="vstup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="T_ukoncit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Ukoncit</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="vystup">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Vstupni text</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Vystupni text</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>571</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="vypis_tabulky">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>561</width>
      <height>191</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>111</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>15</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>37</number>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabulka</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="klicvstup">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Zadej klic</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
